tests/functional/api/test_groups.py

```diff
@@ -28,6 +28,16 @@ def test_it_returns_http_200_with_valid_payload(self, app, token_auth_header):
 
         assert res.status_code == 200
 
+    def test_it_ignores_non_whitelisted_fields_in_payload(self, app, token_auth_header):
```

Comment:
FWIW, this woulda passed before the changes in this PR, but I wanted an explicit test.